esrally/driver/runner.py

```diff
@@ -23,6 +23,8 @@
 from collections import Counter, OrderedDict
 from copy import deepcopy
 
+import elasticsearch
```

Comment:
There is an issue with import this at the top level: it changes logging defaults and the increased logging fills up message queues of the actor system causing serious problems. You'll see that there are several places in this module (and other modules) where `import elasticsearch` is issue right before the place we want to use the exception, e.g. see in [#ForceMerge](https://github.com/elastic/rally/blob/200a830a7505b5275a81a9396514a56cb470bb9c/esrally/driver/runner.py#L570).

So to prevent this, you should `import elasticsearch` inside the `__call__` methods -- as required -- as seen elsewhere.